pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -128,6 +129,9 @@ stdenv.mkDerivation rec {
     cairo
     gtk-mac-integration
   ];
+  propagatedBuildInputs = [
+    ghostscript
```

Comment:
Propagation will not add it to `PATH`. You want a wrapper or patching the executable path in the code.

Reply:
I'll be honest that I don't understand why, but I tested this and it did happen to work... (adding just to the `buildInputs` was not sufficient)